### DISCUSSION TITLE
Top bar doesn't respect workspace color customisations

### DISCUSSION THREAD

**Discussion by SimeonC** [sapegin/squirrelsong#22] [Ideas]
![CleanShot 2024-07-10 at 16 08 42](https://github.com/sapegin/squirrelsong/assets/1085899/ce891469-2705-495f-9fd3-b259f8ecc544)

The background of some elements appears to be gray when hovered and the "search bar" appears as white instead of the set color

## Workspace settings file;
```json
{
  "folders": [
    {
      "path": "../omitted"
    }
  ],
  "settings": {
    "workbench.colorCustomizations": {
      "titleBar.activeForeground": "#FFF",
      "titleBar.inactiveForeground": "#000000CC",
      "titleBar.activeBackground": "#ff3838",
      "titleBar.inactiveBackground": "#ff535379"
    }
  }
}
```

**Comment by sapegin** [comment]
That's by design: most colors are solid to make sure they match the palette.
